src/action/createCDAToken.ts

```diff
@@ -1,32 +1,43 @@
-import type { Environment, Space } from "contentful-management";
-import { Logger } from "../utils";
+import type { Environment, Space } from 'contentful-management';
+import { Logger } from '../utils';
 
-export default async function ({ space, tokenKeyName, environment }: { space: Space, tokenKeyName: string, environment: Environment }) {
-    const spaceKeys = await space.getApiKeys();
+export default async function ({
+  space,
+  tokenKeyName,
+  environment,
+}: {
+  space: Space;
+  tokenKeyName: string;
+  environment: Environment;
+}) {
+  const spaceKeys = await space.getApiKeys();
 
-    const exists = spaceKeys.items.some(item => item.name === tokenKeyName)
+  const exists = spaceKeys.items.some((item) => item.name === tokenKeyName);
 
-    if (exists) {
-        Logger.log(`CDA token ${tokenKeyName} is already created`);
-    } else {
-        Logger.log(`Creating new CDA token "${tokenKeyName}" for ephemeral environment "${environment.sys.id}"...`);
+  if (exists) {
+    Logger.log(`CDA token ${tokenKeyName} is already created`);
+  } else {
+    Logger.log(`Creating new CDA token "${tokenKeyName}" for ephemeral environment "${environment.sys.id}"...`);
 
-        try {
-            const key = await space.createApiKey({
-                name: tokenKeyName,
-                environments: [{
-                    sys: {
-                        type: "Link",
-                        linkType: "Environment",
-                        id: environment.sys.id,
-                    },
-                }],
-            })
+    try {
+      // Olex QQ: ... new key never returned / used?
```

Comment:
Used for service access. We don't need it during the migration, it's more for keeping the same access if, say, we clone something from staging and update.